universal-application-tool-0.0.1/tailwind.config.js

```diff
@@ -1,42 +1,105 @@
+const fs = require('fs')
+
+// For stylesDict from Styles.java and ReferenceClasses.java
+const RGX_KEY = /(?<= +public +static +final +String +)([0-9A-Z_]+)/g
+const RGX_VAL =
+  /(?<= +public +static +final +String +[0-9A-Z_]+ += +")([a-z0-9-/.]+)/g
+
+const STYLE_RGX = /(?<=(Styles|ReferenceClasses)\.)([0-9A-Z_]+)/g
+
+const PREFIXES = [
+  'even',
+  'focus',
+  'focus-within',
+  'hover',
+  'disabled',
+  'sm',
+  'md',
+  'lg',
+  'xl',
+  '2xl',
+]
+
+// Used to read Styles.java and ReferenceClasses.java to get dictionary mapping
+// for all possible base styles (no prefixes)
+function addStyleDictMatches(matches, file_contents) {
+  let count = 1
```

Comment:
Suggestion: rename to line_number, set to 0 and do ++line_number as the first line in the for.